packages/did-ipfs-store/src/didStore.ts

```diff
@@ -48,6 +67,36 @@ export class DidStore implements IDidStore {
     }
   }
 
+  /**
+   * Checks if file is pinned on cluster
+   * @param cid CID
+   * @param isFullyReplicated specifies whether to check if data is fully replicated or pinned at least on one node
+   */
+  async isPinned(cid: string, isfullyReplicated = true): Promise<boolean> {
+    const repication_factor_min = isfullyReplicated
+      ? (await this.allocations(cid)).replication_factor_min
+      : REPLICATION_FACTOR_MIN;
```

Comment:
- Is this `clusterNodeCount` variable name that I've suggested accurate @JGiter ?
- why the snake case 🐍  on `repication_factor_min`?
- `repication_factor_min` is a confusing variable name because it is the same as `REPLICATION_FACTOR_MIN` but lowercase
```suggestion
    const clusterNodeCount = (await this.allocations(cid)).replication_factor_min;
    const repication_factor_min = isfullyReplicated
      ? clusterNodeCount
      : REPLICATION_FACTOR_MIN;
```

Reply:
snake case variables are returned from cluster
`clusterNodeCount` isn't accurate either. Let it be just `nodes`
Agree on confusing `REPLICATION_FACTOR_MIN`. `isfullyReplicated` as I can see now is also not precise. Probably the best in this case is to mirror cluster configuration option by `replication` option which takes values from {LOCAL, MIN, MAX} enum